<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="backgroundLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-image: url(background.jpg);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="loginLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="avatarLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    border-radius: 50px;
    background-color: white;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 1px solid #DDDDDD;
    border-radius: 4px;
    padding: 5px;
    font-size: 14px;
    margin: 10px;
}</string>
        </property>
        <property name="placeholderText">
         <string>账号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 1px solid #DDDDDD;
    border-radius: 4px;
    padding: 5px;
    font-size: 14px;
    margin: 10px;
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="checkboxLayout">
        <item>
         <widget class="QCheckBox" name="rememberPwdCheck">
          <property name="text">
           <string>记住密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="autoLoginCheck">
          <property name="text">
           <string>自动登录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #12B7F5;
    color: white;
    border: none;
    border-radius: 4px;
    font-size: 16px;
    margin: 10px;
}
QPushButton:hover {
    background-color: #0EA2E4;
}
QPushButton:pressed {
    background-color: #0B96D9;
}</string>
        </property>
        <property name="text">
         <string>登 录</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="linksLayout">
        <item>
         <widget class="QLabel" name="registerLink">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #12B7F5;
}</string>
          </property>
          <property name="text">
           <string>注册账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="forgotPwdLink">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: #12B7F5;
}</string>
          </property>
          <property name="text">
           <string>找回密码</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>